Ginger/Ginger/SolutionWindows/TreeViewItems/BusinessFlowTreeItem.cs

```diff
@@ -118,6 +132,7 @@ void ITreeViewItem.SetTools(ITreeView TV)
                 TreeViewUtils.AddSubMenuItem(ExportMenu, "Export to ALM", ExportToALM, null, eImageType.ALM);
                 TreeViewUtils.AddSubMenuItem(ExportMenu, "Map to ALM", MapToALM, null, eImageType.MapALM);
                 TreeViewUtils.AddSubMenuItem(ExportMenu, "Export to CSV", ExportToCSV, null, eImageType.CSV);
+                TreeViewUtils.AddSubMenuItem(ExportMenu, "Export to BPMN file", ExportBPMNMenuItem_Click, icon: eImageType.ShareExternal);
```

Comment:
As we are creating ZIP file, please change name to 'Export as Otoma Use Case Zip file'